commands/local/export.js

```diff
@@ -0,0 +1,59 @@
+module.exports.command = 'export <codemod-name>';
+module.exports.desc = 'Export the current transform to ast-explorer';
+
+module.exports.builder = function builder(yargs) {
+  yargs.positional('codemod-name', {
+    describe: 'the name of the codemod to export',
+  });
+};
+
+module.exports.handler = function handler(options) {
+  const fs = require('fs-extra');
+
+  let { codemodName } = options;
+
+  const Octokit = require('@octokit/rest');
+  const octokit = new Octokit({ auth: 'acabfddbefb244cb3e674fa84046a907c78f2294' });
```

Comment:
🤔 - Seems odd to hard code this here? Is this a mistake?

Reply:
Yep, my bad, will remove that.